libpod/container_internal_linux.go

```diff
@@ -2225,6 +2249,92 @@ func (c *Container) copyOwnerAndPerms(source, dest string) error {
 	return nil
 }
 
+// harmlessHostPath validates if a host path is harmless and can be modified
+func (c *Container) harmlessHostPath(path string) bool {
+	excludePaths := map[string]bool{
+		"/":           true,
+		"/bin":        true,
+		"/boot":       true,
+		"/dev":        true,
+		"/etc":        true,
+		"/etc/passwd": true,
+		"/etc/pki":    true,
+		"/etc/shadow": true,
+		"/home":       true,
+		"/lib":        true,
+		"/lib64":      true,
+		"/media":      true,
+		"/opt":        true,
+		"/proc":       true,
+		"/root":       true,
+		"/run":        true,
+		"/sbin":       true,
+		"/srv":        true,
+		"/sys":        true,
+		"/tmp":        true,
+		"/usr":        true,
+		"/var":        true,
+		"/var/lib":    true,
+		"/var/log":    true,
+	}
+
+	if home := os.Getenv("HOME"); home != "" {
```

Comment:
Can you use
storage/pkg/homedir.Get() here?

Reply:
This has been changed in containers/common